extension/chrome/elements/compose-modules/compose-render-module.ts

```diff
@@ -192,7 +193,7 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
   };
 
   public closeMsg = () => {
-    $('body').attr('data-test-state', 'closed'); // used by automated tests
+    document.querySelector('body')?.setAttribute('data-test-state', 'closed'); // used by automated tests
```

Comment:
@sosnovsky What was the reason to use javascript query selector instead of jquery?
Was there some problem?

Reply:
I've tried it as possible solution for puppeteer not detecting correct value of `data-test-state` value, but it didn't help. However I left `querySelector` as for such simple selectors we can use native js functions, instead of jquery selectors.

Reply:
I see

Reply:
Initially I thought that it's possible to migrate from jquery to using native js methods, but we have a lot of jquery related code and it'll be too time-consuming. For now we can just start using native js selectors, to not produce more jquery dependent code.